src/cpu/o3/BaseO3CPU.py

```diff
@@ -58,6 +61,31 @@ class CommitPolicy(ScopedEnum):
     vals = ["RoundRobin", "OldestReady"]
 
 
+class SSITIndexingPolicy(SimObject):
+    type = "SSITIndexingPolicy"
+    abstract = True
+    cxx_class = "gem5::IndexingPolicyTemplate<gem5::SSITTagTypes>"
+    cxx_header = "cpu/o3/store_set.hh"
+    cxx_template_params = ["class Types"]
+
+    # Get the associativity
+    assoc = Param.Int(Parent.assoc, "associativity")
+
+
+class SSITSetAssociative(SSITIndexingPolicy):
+    type = "SSITSetAssociative"
+    cxx_class = "gem5::SSITSetAssociative"
+    cxx_header = "cpu/o3/store_set.hh"
+
+    # Get the number of entries in the SSIT from the parent
+    num_entries = Param.Unsigned(
+        Parent.numEntries, "Number of entries in the SSIT"
```

Comment:
I grepped numEntries in python files in src/ and I could not find anything. Does it exist somewhere?

Reply:
Not really, I am manually setting num_entries to SSIT entries. I can change the default to that if you think so.

Reply:
Yeah, we should not use nonexistent variables

Reply:
This is updated.